public/js/app.js

```diff
@@ -1,4 +1,30 @@
 var App = {};
 $(function(){
-  return new App.RequestsListView().render();  
+  var path = window.location.pathname || "",
+  id, model;
+  // replace last trailing /
+  path = path.replace(/\/$/, "");
+  path = path.split('/');
+  id = _.last(path);
+  
+  // Always have requestsListView as the bottom view in the stack
+  var listView = new App.RequestsListView().render();
+  
+  if (!path || !id || id.length !== 24){
```

Comment:
is this a magic number? is it worth to comment when the length comes from? (or extract as a variable)